integr8ly/integreatly-operator
# Oo service monitors

# Issue link
Replaces https://github.com/valerymo/integreatly-operator/pull/29

# What


# Verification steps
Run this branch with the following steps
INSTALLATION_TYPE=managed-api make cluster/prepare/local
INSTALLATION_TYPE=managed-api make code/run
You may need to do the image pull workaround

Verify that there are Service Monitors present in the observability prometheus 
There is a related pr which needs to merge -> https://github.com/redhat-developer/observability-operator/pull/89
